decidim-elections/app/assets/javascripts/decidim/votings/polling_officer_zone/new-closure.js.es6

```diff
@@ -0,0 +1,17 @@
+$(() => {
+  $("#submit-verify-votes").addClass("disabled");
+
+  $("#envelopes_result_total_ballots_count").on("keyup", function() {
+    $("#modal-total-ballots-value").html($("#envelopes_result_total_ballots_count").val());
+  });
+
+  $("#envelopes_result_polling_officer_notes").on("keyup", function() {
+    let modalPollingOfficerNotes = $("#envelopes_result_polling_officer_notes").val()
+
+    if (modalPollingOfficerNotes.length > 0) {
+      $("#btn-submit-from-modal").removeClass("disabled");
+    } else {
+      $("#btn-submit-from-modal").addClass("disabled");
+    }
```

Comment:
💬  You can simplify this with:
```suggestion
$("#btn-submit-from-modal").toggleClass("disabled", !modalPollingOfficerNotes.trim());
```
(Notice the input trimming to prevent the user form submitting empty text). This works because `""` is a falsey value in JS.